apps/src/templates/projects/publishDialog/PublishDialog.jsx

```diff
@@ -54,6 +90,7 @@ class PublishDialog extends Component {
         <div style={{marginBottom: 10}}>
           {i18n.publishToPublicGalleryWarning()}
         </div>
+        {errorMessage && <div style={styles.error}>{errorMessage}</div>}
```

Comment:
For teachers of younger students, I can imagine wanting more information in some of these cases, particularly for the new upload feature. It's likely that students won't know or remember that they agreed to something, so it might be helpful to give the teacher a support article that explains how they can undo this by resetting version history.
It's also something we could address later if/when we started getting tickets about it. I guess this is primarily a question for @moneppo.

Reply:
In any case, I think this is a fine next step and my question shouldn't hold up this PR. :)

Reply:
Good call out. Since restricting publish is behind a feature flag I think we can add a support link later. I'll create a jira task for this.

Reply:
I created a Jira task for the [support article](https://codedotorg.atlassian.net/browse/SL-606) and for [adding the link](https://codedotorg.atlassian.net/browse/SL-605) once the article exists

Reply:
Thanks Molly!